common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -4564,6 +4564,9 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
         "The meaning of this parameter is the inverse of the number of time ticks (cache\n" +
         " operations, currently) that cause the combined recency-frequency of a block in cache\n" +
         " to be halved."),
+    LLAP_LRFU_CUTOFF_PERCENTAGE("hive.llap.io.lrfu.cutoff.percentage", 0.10f,
```

Comment:
Naming here could make a connection with the API I think: I'm missing "hot buffers"
Something like hive.llap.io.lrfu.hotbuffers.cutoff.percentage or hive.llap.io.lrfu.hotbuffers.percentage just so that we use similar terminology in conf name and code.

Reply:
Done.